packages/webapp-libs/webapp-api-client/src/tests/factories/pagination.ts

```diff
@@ -0,0 +1,50 @@
+import { gql } from '@apollo/client';
+import { TypedDocumentNode } from '@graphql-typed-document-node/core';
+import { PageInfo, composeMockedPaginatedListQueryResult, createDeepFactory, makeId } from '../utils';
+import { PaginationListTestQueryQuery, PaginationListTestQueryQueryVariables } from '@sb/webapp-api-client/graphql';
+
+
+export const paginationListTestQuery: TypedDocumentNode<
```

Comment:
To be consistent with the other queries I would move this to the `webapp-api-client/src/hooks/usePaginationQuery/__tests__/usePaginationQuery.graphql.ts`